quickstarts/redmine/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: redmine
+description: |
+  ## Comprehensive monitoring quickstart for Redmine
+  We can monitor your Redmine application in a complex environment and keep you on top of performance with our Ruby agent.
```

Comment:
"You can monitor the performance of your Ruby-on-Rails Redmine app in a complex environment with our Ruby APM agent."